client/src/rpc/merkle_tree.rs

```diff
@@ -27,6 +27,7 @@ pub enum MerkleTreeExtError {
 /// fetching Merkle trees.
 #[async_trait]
 pub trait MerkleTreeExt: RpcConnection {
+    // TODO: add v2 state tree
```

Comment:
I'll look at this later today as part of the [test_indexer](https://github.com/Lightprotocol/light-protocol/pull/1376) refactoring